google-chart.html

```diff
@@ -197,6 +199,25 @@
           this.options = {};
           this.rows = [];
           this.dataTable = null;
+
+          var head = document.querySelector('head');
+          var element = this;
+          var observer = new MutationObserver(function(mutations) {
+              mutations.forEach(function(mutation) {
+                for (var i = 0; i < mutation.addedNodes.length; i++) {
+                  var newNode = mutation.addedNodes[i];
+                  if (newNode.rel && newNode.rel == 'stylesheet') {
+                    //Add the stylesheet as an @import
+                    var cssImport = document.createElement('style');
+                    cssImport.innerHTML = "@import '" + newNode.href + "';";
+                    element.shadowRoot.appendChild(cssImport);
```

Comment:
@philipwalton wdyt about this approach instead of your own PR that ads the styles internal to the shadow dom. What I like about this (despite being a bit hacky) is that the styles will always be up to date from the server.